onnx/defs/math/defs.cc

```diff
@@ -1046,4 +1046,30 @@ ONNX_OPERATOR_SET_SCHEMA(
             "Constrain input and output types to float tensors.")
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput));
 
+static const char* Expand_ver8_doc = R"DOC(
+Broadcast the input tensor following the given shape and the broadcast rule.
+The broadcast rule is similar to numpy.array(input) * numpy.ones(shape):
```

Comment:
You can explicitly say that this operator is similar to numpy.broadcast_to, and describe the difference here.

Reply:
sure :)